docs/reference/API/attr_to_event.md

```diff
@@ -0,0 +1,89 @@
+# List of PyScript Attributes to Events:
+
+PyScript provides a convenient syntax for mapping JavaScript events to PyScript events, making it easy to connect events to HTML tags.
+
+For example, you can use the following code to connect the click event to a button:
+
+```
+<button id="my-button" py-onClick="foo()">Click me</button>
+```
+
+Here is a list of all the available event mappings:
+
+| PyScript Event Name | DOM Event Name |
+|-------------------|----------------|
+| pys-onClick       | click          |
+| pys-onKeyDown     | keydown        |
```

Comment:
These are deprecated, we shouldn't include it (they will be removed on the next release)